msg_test.go

```diff
@@ -122,41 +123,39 @@ func TestUnPackExtendedRcode(t *testing.T) {
 }
 
 func TestUnpackDomainName(t *testing.T) {
+	// manyPointers unpacks perfectly validly to "example.com." after following
+	// 200 pointers. This trips our maxCompressionPointers check as we refuse
```

Comment:
yeah, see prev. comment, who are we to impose limits like this.

Reply:
Will remove in a future PR.